integration/spark/src/test/scala/org/apache/carbondata/spark/testsuite/badrecordloger/BadRecordActionTest.scala

```diff
@@ -270,6 +274,53 @@ class BadRecordActionTest extends QueryTest {
     }
   }
 
+  test("test bad record FAIL with invalid timestamp range") {
+    val csvPath = s"$resourcesPath/badrecords/invalidTimeStampRange.csv"
+    val rows1 = new ListBuffer[Array[String]]
+    rows1 += Array("ID", "date", "time")
+    rows1 += Array("1", "2016-7-24", "342016-7-24 01:02:30")
+    createCSV(rows1, csvPath)
+    sql("DROP TABLE IF EXISTS test_time")
+    sql("""
```

Comment:
can format create SQL in just two lines, as we do not worry about scala style in test files. We can avoid duplicate lines or more lines